src/partial_comparable.cr

```diff
@@ -62,5 +62,18 @@ module PartialComparable(T)
     end
   end
 
+  # The comparison operator.
+  #
+  # Returns a negative number if the object
```

Comment:
Minor detail, but that's an awkward line break.

Reply:
`PartialComparable` is going to be removed anyway.